doc/docusaurus/docs/essential-concepts/versions.md

```diff
@@ -0,0 +1,68 @@
+  This will also change, as we plan to make all Plutus Core versions compatible with all ledger language versions.
+
+When do we introduce a new ledger language version?
+A new ledger language version must be introduced when a new ledger era is rolled out, because in a new ledger era, transactions will be different and contain new and/or modified fields, such as governance actions introduced in the Conway era.
+As a result, new information will need to be passed as arguments to Plutus validators.
+This necessitates a new ledger language version, because the arguments passed to existing ledger language versions must stay exactly the same, so that existing validators continue to work.
+
+Previously it was also the case that introducing new builtin functions or new Plutus Core versions necessitate a new ledger language version, but this is no longer the case.
+Introducing a new ledger language version is very expensive as it must be maintained forever.
+Going forward we expect the launch of new ledger eras to be the only reason for introducing new ledger language versions.
+
+## Plutus Core Version
+
+Plutus Core version is the usual sense of version pertaining to programming languages - in this instance the Plutus Core language.
+So far there have been two Plutus Core versions: 1.0.0 and 1.1.0.
+1.1.0 adds sums-of-products to the language by introducing two new AST node types: `Case` and `Constr`.
+See CIP-85 for more details.
+
+Note that adding new builtin functions does not require a new Plutus Core version.
+Once a new builtin function is added, one can simply start using the new builtin function with an existing Plutus Core version.
+
+## Builtin Semantics Variant
+
+Depending on the protocol version or the ledger language version, we may want to have different behavior of a particular builtin function.
+For example, we might want to fix a bug but need to retain the old buggy behavior for old evaluations, so we have to have a way of knowing which regime we are in.
+
+One example is the builtin function `consByteString`.
+If the first argument is outside of the range of `Word8`, the original behavior is to wrap it around.
+The new behavior starting at Plutus V3 is to throw an exception.
+
+When evaluating a standalone program using the `uplc` executable, flag `-S` or `--builtin-semantics-variant` can be used to inform the evaluator which semantics variant to use.
+
+## Plutus Haskell Package Version
+
+Several Plutus components are regularly released as libraries, such as `plutus-core`, `plutus-ledger-api` and `plutus-tx-plugin`.
+These packages are published on the [Cardano Haskell Package repository](https://github.com/IntersectMBO/cardano-haskell-packages) (CHaP), rather than Hackage, Haskell's default package archive.
+Each release has a version, and this is completely orthogonal and irrelevant to all other aformentioned notions of version.
```

Comment:
suggestion:

"Each release has a version followign a release versioning scheme, completely orthogonal and irrelevant to all other aformentioned notions of version."